src/command/cli.rs

```diff
@@ -34,6 +35,8 @@ impl Cli {
             .about("post quantum blockchain")
             .subcommand(App::new("printchain").about("print all the chain blocks"))
             .subcommand(App::new("createwallet").about("create a wallet"))
+                .about("select encryption method")
+                .arg(Arg::from_usage("<encryption> 'select the encryption method from ecdsa or fndsa'"))
```

Comment:
encryption 引数を付ける場所は `createwallet` コマンドのみだと思いますがあってますか？

```suggestion
            .subcommand(App::new("createwallet").about("create a wallet")
                .about("select encryption method")
                .arg(Arg::from_usage("<encryption> 'select the encryption method from ecdsa or fndsa'")))
```

Reply:
上記が正しければ `.about("select encryption method")` は `from_usage` と重複するのでいらなさそうですね

```suggestion
            .subcommand(App::new("createwallet").about("create a wallet")
                .arg(Arg::from_usage("<encryption> 'select the encryption method from ecdsa or fndsa'")))
```

Reply:
あと `FNDSA` をデフォルト文字列として渡してあげちゃってもいいかもです

Reply:
指定してない場合FNDSAが勝手に選ばれるという認識